From: Hideyuki Yamashita <yamashita.hideyuki@ntt-tx.co.jp>
Date: Mon, 6 Jan 2020 11:04:35 +0900
Subject: [PATCH 7/9] docs: add install drivers for Mellanox NIC

This patch is to add description of installing drivers of Mellanox NIC.

Signed-off-by: Hideyuki Yamashita <yamashita.hideyuki@ntt-tx.co.jp>
Signed-off-by: Yasufumi Ogawa <yasufum.o@gmail.com>
---
 docs/guides/gsg/setup.rst | 12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/docs/guides/gsg/setup.rst b/docs/guides/gsg/setup.rst
index 9bd30ef..4769e13 100644
--- a/docs/guides/gsg/setup.rst
+++ b/docs/guides/gsg/setup.rst
@@ -285,6 +285,18 @@ Python 2 or 3 ?
 Without SPP container tools, Python2 is not supported anymore.
 SPP container will also be updated to Python3.
 
+Drivers for Mellanox NIC
+------------------------
+
+In case of using MLX5 NIC, you have to install Driver.
+You can download driver from Mellanox's `SW/Drivers
+<https://www.mellanox.com/page/mlnx_ofed_matrix?mtag=linux_sw_drivers>`.
+The following example assumes that MLNX_OFED_LINUX-4.7-1.0.0.1-ubuntu18.04-x86_64.tgz is downloaded.
+
+.. code-block:: console
+
+   $cd MLNX_OFED_LINUX-4.7-1.0.0.1-ubuntu18.04-x86_64/
+   $sudo ./mlnxofedinstall --upstream-libs --dpdk --force
 
 Reference
 ---------
-- 
2.18.0

